docs/tutorials/delete-a-user-levibeverly.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: page
+---
+
+# Tutorial: Delete a user
+
+Use the following steps to quickly identify and delete a user from your subscriber base. You can perform these steps using [cURL commands](#delete-a-user-with-curl-commands) or in the [Postman application](#delete-a-user-in-postman).
+
+*Note: This tutorial assumes that you have already completed the To-Do Service first-time setup process. If you have not, complete the steps described in [First-time setup](), then return to this tutorial.*
+
```

Comment:
Maybe add a note about the preferred command line tool here and mention that some command syntax might need to be adapted if you're using another command line tool.